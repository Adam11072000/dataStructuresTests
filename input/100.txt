init 147
quit