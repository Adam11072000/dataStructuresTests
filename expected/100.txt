init done.
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: FAILURE
quit done.
